sdk/template/Azure.Template/src/TemplateClient.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.Template.Models;
+
+namespace Azure.Template
+{
+    /// <summary> The Template service client. </summary>
+    public partial class TemplateClient
+    {
+        /// <summary> The GET operation is applicable to any secret stored in Azure Key Vault. This operation requires the secrets/get permission. </summary>
+        /// <param name="secretName"> The name of the secret. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual async Task<Response<SecretBundle>> GetSecretValueAsync(string secretName, CancellationToken cancellationToken = default)
+        {
+            using var scope = ClientDiagnostics.CreateScope("TemplateClient.GetSecretValue");
+            scope.Start();
+
+            try
+            {
+                RequestContext context = new RequestContext()
+                {
+                    CancellationToken = cancellationToken,
+                };
+                Response response = await GetSecretAsync(secretName, context).ConfigureAwait(false);
+                SecretBundle value = SecretBundle.FromResponse(response);
+                return Response.FromValue(value, response);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary> The GET operation is applicable to any secret stored in Azure Key Vault. This operation requires the secrets/get permission. </summary>
+        /// <param name="secretName"> The name of the secret. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual Response<SecretBundle> GetSecretValue(string secretName, CancellationToken cancellationToken = default)
+        {
+            using var scope = ClientDiagnostics.CreateScope("TemplateClient.GetSecretValue");
```

Comment:
Seems in our code, we use a combination of 'var' and actual type when declare a varible: 
     var scope
     RequestContext context 
Do we have a guidance about what pattern to use and shall we make the code consistent for this?

Reply:
Ah, @heaths  confirms this here: https://github.com/Azure/azure-sdk-for-net/pull/28016#discussion_r850879368

Reply:
resolved

Reply:
FWIW, it's actually in our style settings: https://github.com/Azure/azure-sdk-for-net/blob/951e011c5423c2e427a4068a3281f314f4c8c4c9/.editorconfig#L42-L45